README.md

```diff
@@ -254,7 +254,7 @@ If you like these colors and want some alternatives, you can find them on https:
 
 If you want to run this repo locally after cloning:
 
-```
+```bash
```

Comment:
Can you also change the README inside the gatsby-theme-help-center repo with the same changes? A quick `cp` over would work.

I have to put the README in two places because the other one is published to npm and I had weird problems when trying to publish to npm while having the README using a symbolic link from the root.

If you don't have time let me know and I can do it myself :)

Reply:
Done :)

Reply:
Great! I'll merge and publish this tomorrow, thanks for contributing :)